test/TestUtilities/Test.Utility/DependencyResolver/DependencyProvider.cs

```diff
@@ -154,6 +154,7 @@ public TestPackage DependsOn(string id, string version, LibraryDependencyTarget
                     },
                     VersionCentrallyManaged = versionCentrallyManaged,
                     ReferenceType = libraryDependencyReferenceType,
+                    SuppressParent = privateAssets ?? LibraryIncludeFlagUtils.DefaultSuppressParent,
```

Comment:
Thinking we shouldn't set `SuppressParent` unless a privateAssets was specified. 
If we specify the default in the test, then `LibraryDependency` changes the default property value, the test won't be accurate, right?

Reply:
I agree, I've also done the same for the `libraryDependencyReferenceType`.